translations/ar/ar_test.go

```diff
@@ -1,705 +1,622 @@
-		},
-		{
-			ns:       "Test.UniqueArray",
-			expected: "يجب أن يحتوي UniqueArray على قيم فريدة",
-		},
-		{
-			ns:       "Test.UniqueMap",
-			expected: "يجب أن يحتوي UniqueMap على قيم فريدة",
-		},
-		{
-			ns:       "Test.JSONString",
-			expected: "يجب أن يكون JSONString نص json صالح",
-		},
-		{
-			ns:       "Test.JWTString",
-			expected: "يجب أن يكون JWTString نص jwt صالح",
-		},
-		{
-			ns:       "Test.LowercaseString",
-			expected: "يجب أن يكون LowercaseString نص حروف صغيرة",
-		},
-		{
-			ns:       "Test.UppercaseString",
-			expected: "يجب أن يكون UppercaseString نص حروف كبيرة",
-		},
-		{
-			ns:       "Test.Datetime",
-			expected: "لا يتطابق Datetime مع تنسيق 2006-01-02",
-		},
-		{
-			ns:       "Test.PostCode",
-			expected: "لا يتطابق PostCode مع تنسيق الرمز البريدي للبلد SG",
-		},
-		{
-			ns:       "Test.PostCodeByField",
-			expected: "لا يتطابق PostCodeByField مع تنسيق الرمز البريدي للبلد في حقل PostCodeCountry",
-		},
-		{
-			ns: "Test.Image",
-			expected: "يجب أن تكون Image صورة صالحة",
-		},
+	assert.Equal(t, nil, err)
+
+	return validate, trans
+}
+
+// TestBasicTranslations tests simple validation tags
+func TestBasicTranslations(t *testing.T) {
+	type TestStruct struct {
+		Name     string `validate:"required"`
+		Email    string `validate:"required,email"`
+		Age      int    `validate:"required,gte=18"`
+		Password string `validate:"required,min=8"`
+	}
+
+	validate, trans := setupTranslator(t)
+
+	// Test required field
+	testObj := TestStruct{
+		Email:    "test@example.com",
+		Age:      20,
+		Password: "password123",
+	}
+
+	err := validate.Struct(testObj)
+	assert.NotEqual(t, nil, err)
+
+	errs := err.(validator.ValidationErrors)
+	assert.Equal(t, "حقل Name مطلوب", errs[0].Translate(trans))
+
+	// Test email validation
+	testObj = TestStruct{
+		Name:     "Test User",
+		Email:    "invalid-email",
+		Age:      20,
+		Password: "password123",
+	}
+
+	err = validate.Struct(testObj)
+	assert.NotEqual(t, nil, err)
+
+	errs = err.(validator.ValidationErrors)
+	assert.Equal(t, "يجب أن يكون Email عنوان بريد إلكتروني صالح", errs[0].Translate(trans))
+
+	// Test gte validation
+	testObj = TestStruct{
+		Name:     "Test User",
+		Email:    "test@example.com",
+		Age:      16,
+		Password: "password123",
+	}
+
+	err = validate.Struct(testObj)
+	assert.NotEqual(t, nil, err)
+
+	errs = err.(validator.ValidationErrors)
+	assert.Equal(t, "يجب أن يكون طول Age على الأقل 18 أحرف", errs[0].Translate(trans))
+
+	// Test min validation
+	testObj = TestStruct{
+		Name:     "Test User",
+		Email:    "test@example.com",
+		Age:      20,
+		Password: "pass",
+	}
+
+	err = validate.Struct(testObj)
+	assert.NotEqual(t, nil, err)
+
+	errs = err.(validator.ValidationErrors)
+	translation := errs[0].Translate(trans)
+	assert.Equal(t, true, contains(translation, "Password يجب أن يكون 8 أحرف على الأقل"))
+}
+
+// Helper function to check if a string contains another string
+func contains(s, substr string) bool {
```

Comment:
Can we use `strings.Contains()` instead of this helper function?
https://pkg.go.dev/strings#Contains